### PR TITLE
Atomic asserts improvments

### PR DESCRIPTION
**Pull request by koonopek**
1. created `atomic-asset-test` package to share common tests for typescript and rust implementation
2. Use same error messages for typescript and rust
3.  Add `owner` method
4. Update warp to 1.2.55
5. changed build/bundling to work as isntalled packages
6. use peerDependency were possible

### CODE REVIEW

File: contracts/atomic-asset-test/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "atomic-asset-test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "private": false,
+  "exports": {
```
Comment (asiaziola) [inline contracts/atomic-asset-test/package.json:7]: are you sure about these exports? :)

File: contracts/atomic-asset-test/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "contract": "./dist/contracts/atomic-asset-typescript/contract/atomic-asset.js",
+    "lib": "./dist/contracts/atomic-asset-typescript/lib/index.js"
+  },
+  "scripts": {
+    "test": "jest --config jest.config.js",
+    "build": "tsc --project tsconfig-build.json"
```
Comment (asiaziola) [inline contracts/atomic-asset-test/package.json:13]: is this script necessary for this package?

File: test/package.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
```
[the remark this answers is not in the document]
Comment (koonopek) [reply test/package.json:1]: uhhh, miss click :)

File: contracts/atomic-asset-test/package.json (hunk 1 above)
Comment (koonopek) [reply contracts/atomic-asset-test/package.json:7]: Whoops thanks :)

File: contracts/atomic-asset-test/package.json (hunk 2 above)
Comment (koonopek) [reply contracts/atomic-asset-test/package.json:13]: Of course not :)

File: examples/staking/scripts/deployERC20.js [hunk 4]
```diff
@@ -23,7 +23,7 @@ const { contractSrc, wasmGlueCode, wasmSrcCodeDir } = require('erc20/utils');
     evolve: '',
   };
 
-  let deployment = await warp.createContract.deploy({
+  let deployment = await warp.deploy({
     wallet: ownerWallet,
```
[the remark this answers is not in the document]
Comment (koonopek) [reply examples/staking/scripts/deployERC20.js:27]: Ohh no, I haven't testes this scripts will do!
Also why we are not wraping into ArweaveSigner in SDK (only if bundling)?

**Pull request merged**